lambdas/validatecricheck/src/main/java/uk/gov/di/ipv/core/validatecricheck/validation/FraudEvidenceValidator.java

```diff
@@ -0,0 +1,22 @@
+package uk.gov.di.ipv.core.validatecricheck.validation;
+
+import com.google.gson.Gson;
+import com.nimbusds.jose.shaded.json.JSONArray;
+import uk.gov.di.ipv.core.validatecricheck.domain.FraudEvidence;
+
+public class FraudEvidenceValidator implements CriEvidenceValidator {
+    public static final int ONLY_ELEMENT = 0;
+    public static final int GPG_45_M1A_FRAUD_SCORE = 1;
+    private static final Gson gson = new Gson();
+
+    @Override
+    public boolean validate(JSONArray evidenceArray) {
+        FraudEvidence evidence =
```

Comment:
Just want to double check here, but will GSON be able to correctly convert this if the ci property isn't there?

Reply:
Yep - there's a test for that scenario here: https://github.com/alphagov/di-ipv-core-back/pull/290/files#diff-ffd5c03bafeb836c40ed264a76fa32747b26f16b90866250e392eccfac2b3c02R75

Although I've realised that the test name is wrong 🙄

Reply:
But isn't that test for if the property exists but is an empty list. I think there is the possibility that the ci property could be missing all together?

Reply:
As discussed there is another test here that uses a signed JWT without a ci property that proves this will work as expected.